### DISCUSSION TITLE
Planting Tool Ideas

### DISCUSSION THREAD

**Discussion by timgrote** [timgrote/Raindrop-Feedback#4] [Ideas]
Let's collect ideas people have for the planting tool, I'll kick it off.

Would it be useful to layout plants along a polyline like we do with sprinklers?
Or something like a 'Fill Area' routine that fills a given areas with plants based on spacing?

**Comment by EJHida10** [comment]
Yes spacing them out especially for street trees would be helpful.

Shrubs the fill command would be a time saver so that 1 doesn't have to lay them all out individualy as well.
